src/super_gradients/training/datasets/detection_datasets/yolo_format_detection.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import imagesize
+import numpy as np
+from typing import List, Optional
+
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.training.datasets.detection_datasets.detection_dataset import DetectionDataset
+from super_gradients.training.exceptions.dataset_exceptions import DatasetValidationException
+
+from super_gradients.training.datasets.data_formats import ConcatenatedTensorFormatConverter
+from super_gradients.training.datasets.data_formats.default_formats import XYXY_LABEL, LABEL_NORMALIZED_CXCYWH
+
+logger = get_logger(__name__)
+
+
+class YoloV5FormattedDetectionDataset(DetectionDataset):
+    """Base dataset to load ANY dataset that is with a similar structure to the YoloV5 dataset.
+
+    **Note**: For compatibility reasons, the dataset returns labels in Coco format (XYXY_LABEL) and NOT in YoloV5 format (LABEL_CXCYWH).
+
+    Output format: XYXY_LABEL (x, y, x, y, class_id)
+    """
+
+    def __init__(
+        self,
+        data_dir: str,
+        images_dir: str,
+        labels_dir: str,
+        classes: List[str],
+        class_ids_to_ignore: Optional[List[int]] = None,
+        *args,
+        **kwargs,
+    ):
+        """
+        :param data_dir:                Where the data is stored.
+        :param images_dir:              Name of the directory that includes all the images. Path relative to data_dir.
```

Comment:
Please call it images_dir_name and labels_dir_name to make it clearer

Reply:
done